include/aws/common/atomics_msvc.inl

```diff
@@ -114,7 +143,8 @@ static inline void aws_atomic_priv_barrier_before(enum aws_memory_order order, e
      * Volatile ops may or may not imply this barrier, depending on the /volatile: switch, but adding an extra
      * barrier doesn't hurt.
      */
-    _ReadWriteBarrier();
+    RW_BARRIER();
```

Comment:
There are a few sentences worth of comments above this explaining exactly why it is the way it is for x86

Edit these so they still make sense with the new code. Maybe move the comments up where these macros are declared? I honestly don't understand all this subtlety, so I leave it to you